packages/pilot/src/components/Event/index.js

```diff
@@ -81,15 +97,16 @@ class Event extends React.Component {
 
         <div className={style.info}>
           {
-            collapsed && childrenIsArray
+            collapsed && isMoreThanOneOperation
               ? head(children)
               : children
           }
         </div>
 
-        <div className={style.indicator}>
+        <div className={cx(style.indicator, { [style.line]: !active })}>
```

Comment:
Este `cx` não está muito explicativo poderia ser trocado por um nome mais explicativo tipo `classNames`.
```suggestion
        <div className={classNames(style.indicator, { [style.line]: !active })}>
```

Reply:
Boa, troquei o nome